src/main.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(unused_must_use)]
+#![deny(rustdoc::broken_intra_doc_links)]
```

Comment:
this can probably be added to the previous `deny` with a comma

Reply:
I think I actually prefer on separate lines. Makes it easier for the diff what do you think?